### DISCUSSION TITLE
Elevation data (30m resolution) for the whole world (including +60°N)

### DISCUSSION THREAD

**Discussion by mach90** [domlysz/BlenderGIS#898] [Ideas]
This gives you elevation data for the whole world including above 60°N which is not provided by the 3 sources available by default.

Proof, below is Isafjordur Iceland
![image](https://github.com/domlysz/BlenderGIS/assets/132974650/cdeb6cce-4c6c-4a64-984f-1b80a68e609d)

# How to
In **BlenderGIS Addon settings** > **Remote data source** > **Elevation server** > **Add**

**URL Template**
https://portal.opentopography.org/API/globaldem?demtype=AW3D30&west={W}&east={E}&south={S}&north={N}&outputFormat=GTiff&API_Key={API_KEY}

**Description**
ALOS World 3D 30m

**Comment by sarnason** [comment]
Wow, thanks. This should be added to the official release.
